dev-lang/crystal/crystal-0.7.7.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
```

Comment:
You didn't use repoman to commit. Please always use it and listen to its advice.